src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -2066,11 +2066,10 @@ HRESULT CEndlessUsbToolDlg::OnLiveUsbClicked(IHTMLElement* pElement)
 {
     FUNCTION_ENTER;
 
-	SetSelectedInstallMethod(InstallMethod_t::LiveUsb);
+    SetSelectedInstallMethod(InstallMethod_t::LiveUsb);
```

Comment:
Maybe it would be possible to avoid this line appearing in the diff by avoiding the spacing changes. Or is this s CR vs. CRLF thing? At any rate, do not consider this thing a blocker — just a “nice to have” thing.